iceoryx_examples/iceperf/iceperf_hardy.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
please remove empty line here

Reply:
add or remove you have to decide what you want ;-)